exp/api/v1beta2/types.go

```diff
@@ -25,11 +25,11 @@ import (
 const (
 	// ExternalResourceGCAnnotation is the name of an annotation that indicates if
 	// external resources should be garbage collected for the cluster.
-	ExternalResourceGCAnnotation = "aws.cluster.x-k8s.io/external-resource-gc"
+	ExternalResourceGCAnnotation = infrav1.ExternalResourceGCAnnotation
```

Comment:
I think this would be duplication, either we keep these constants in exp/api/v1beta2, or in api/v1beta2

Reply:
Now these constants are in `api/v1beta2`, but I kept them as duplicates in `exp/api/v1beta2` to prevent a breaking change in the API. People who used the constants from `exp` can continue to do so, but now the correct location is `api/v1beta2`.

Reply:
I might need more opinions here @richardcase @Skarlso @dlipovetsky

Reply:
It's OK not to duplicate it. If we remove it, make sure this gets into a minor release, though, because it might be a breaking change. Although, honestly, I doubt too many people call this from anywhere outside of CAPA.

Reply:
alright! I'll remove it from `exp` completely since we don't consider it as a breaking change. for sure, it will make things simpler.

Reply:
Noice.